Mathlib/Algebra/Order/GroupWithZero/Unbundled/Basic.lean

```diff
@@ -1138,15 +1180,16 @@ variable [LinearOrder G₀] {a b c d : G₀}
 section PosMulMono
 variable [PosMulMono G₀]
 
-@[simp] lemma inv_neg'' : a⁻¹ < 0 ↔ a < 0 := by
+@[simp] lemma inv_lt_zero : a⁻¹ < 0 ↔ a < 0 := by
```

Comment:
That's a step back IMO. Just a few days ago I was looking for this lemma, typed `inv_neg` and searched through auto-complete for the correct lemma

Reply:
OK, I'll move this change to another PR so that we can discuss it there.